pkgs/servers/mail/dovecot/plugins/fts_xapian/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkg-config, dovecot, libtool, xapian, icu64 }:
+stdenv.mkDerivation {
+  pname = "fts-xapian";
+  version = "1.2.11a";
+
+  src = fetchFromGitHub {
+    owner = "grosjo";
+    repo = "fts-xapian";
+    rev = "1.2.11a";
+    sha256 = "1n9bvv2rgnmmpm472y067yfzwah39ig1ziwyk5gmrbg0k54x81dk";
+  };
+
+  buildInputs = [ dovecot autoconf automake libtool pkg-config xapian icu64 ];
+
+  preConfigure = ''
+    export PANDOC=false
+    autoreconf -vi
+  '';
+
+  configureFlags = [
+    "--with-dovecot=${dovecot}/lib/dovecot"
+    "--without-dovecot-install-dirs"
+    "--with-moduledir=$(out)/lib/dovecot"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/grosjo/fts-xapian;
```

Comment:
URLs should be quoted per RFC45.

https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md